invokeai/app/invocations/latent.py

```diff
@@ -118,6 +118,9 @@ class DenoiseLatentsInvocation(BaseInvocation):
     )
     denoising_start: float = InputField(default=0.0, ge=0, le=1, description=FieldDescriptions.denoising_start)
     denoising_end: float = InputField(default=1.0, ge=0, le=1, description=FieldDescriptions.denoising_end)
+    cfg_rescale_multiplier: float = InputField(
+        default=0, ge=0, lt=1, description=FieldDescriptions.cfg_rescale_multiplier, ui_type=UIType.Float
+    )
```

Comment:
You can safely leave off the `ui_type=UIType.Float` on this field - the client will infer it as a float correctly